homeassistant/components/hikvision/binary_sensor.py

```diff
@@ -67,10 +66,12 @@
     "Entering Region": BinarySensorDeviceClass.MOTION,
 }
 
+_LOGGER = logging.getLogger(__name__)
+
 CUSTOMIZE_SCHEMA = vol.Schema(
     {
-        vol.Optional(CONF_IGNORED, default=DEFAULT_IGNORED): cv.boolean,
-        vol.Optional(CONF_DELAY, default=DEFAULT_DELAY): cv.positive_int,
+        vol.Optional("ignored", default=False): cv.boolean,
+        vol.Optional("delay", default=0): cv.positive_int,
     }
 )
```

Comment:
So before we had this customization schema. I think the functionality of ignored can be compared to it being disabled, but what about delay? Is it still an issue today as described in the docs? "Specify the delay to wait after a sensor event ends before notifying Home Assistant in seconds. This is useful to catch multiple quick trips in one window without the state toggling on and off"